cypress/page/products-list.page.ts

```diff
@@ -1,14 +1,23 @@
+// /<reference types="cypress" />
+
 class ProductsListPage {
   private productToAdd: string;
   private proceedToCheckoutBtnPopUp: string
+  private elementHTML: string;
   
   constructor() {
     this.productToAdd = ".ajax_add_to_cart_button > span";
     this.proceedToCheckoutBtnPopUp = ".button-container > .button-medium > span";
+    this.elementHTML = ".product-container";
   }
   
-  public addProductToCart(): void {
-    cy.get(this.productToAdd).click()
+  findProductByName(prodName: string) {
+    return cy.get(this.elementHTML).filter(`:contains(${prodName})`);
```

Comment:
element HTML is not a proper name for the variable, you should change it to something more descriptive

Reply:
Done... changed for a better named one.